SEED-platform/seed
# Update links for support

#### Any background context you want to provide?
links were out of date

#### What's this PR do?
* new links

#### How should this be manually tested?
* CHECK GRAMMAR/SPELING (sic)
* If this is good, then I'll update the website and buildingdata.energy.gov

#### What are the relevant tickets?
#3659 

#### Screenshots (if appropriate)
